tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py3
+
+[testenv]
+commands =
+    python setup.py test -q
+deps =
+    .[test]
```

Comment:
This file is also missing a trailing newline.

(Possibly it's a minor annoyance rather than not a problem.)